desktop/cmp/input/DateInput.js

```diff
@@ -43,6 +43,9 @@ DateInput.propTypes = {
     ...HoistInputPropTypes,
     value: PT.oneOfType([PT.instanceOf(Date), PT.instanceOf(LocalDate)]),
 
+    /** True to commit on every change/keystroke, default false. */
```

Comment:
I think this doc string will need more elaboration re. any special behaviors in the context of the date input, where the text input might not be valid, etc.